bpf/common/sockaddr.h

```diff
@@ -17,7 +17,8 @@ typedef struct accept_args {
 
 static __always_inline bool parse_sock_info(struct sock *s, connection_info_t *info) {
     short unsigned int skc_family;
-    BPF_CORE_READ_INTO(&skc_family, s, __sk_common.skc_family);
+    int err = BPF_CORE_READ_INTO(&skc_family, s, __sk_common.skc_family);
```

Comment:
I would remove these lines and the commented out struct sock *s...

Reply:
Done in 846d2158